### DISCUSSION TITLE
xarray.DataTree release!

### DISCUSSION THREAD

**Discussion by TomNicholas** [pydata/xarray#9680] [Announcements]
## Xarray [v2024.10.0](https://github.com/pydata/xarray/releases/tag/v2024.10.0) brings official support for [`xarray.DataTree`](https://docs.xarray.dev/en/latest/user-guide/data-structures.html#datatree)! 🍾 

We will write an announcement blog post soon, but in the meantime check out the dedicated [documentation on DataTree](https://docs.xarray.dev/en/latest/user-guide/hierarchical-data.html).

The [original `xarray-contrib/datatree` repository](https://github.com/xarray-contrib/datatree) has now been archived in favour of the implementation in this repository. There is a [migration guide here](https://github.com/pydata/xarray/blob/main/DATATREE_MIGRATION_GUIDE.md).

This was a [huge effort](https://github.com/pydata/xarray/pulls?q=is%3Apr+label%3Atopic-DataTree+is%3Aclosed) that involved many people - shoutout particularly to @owenlittlejohns @flamingbear @eni-awowale @aladinor @keewis and @shoyer for helping with the migration to xarray `main`.

DataTree is still new so there will likely be some bugs lurking - please report them as new issues on this repository. Please also raise an issue if you have a feature request for datatree. And if you think we missed anything in the migration guide. If you have other questions, comments, or feedback please let us know in this discussion down below!

**Comment by allen-adastra** [comment]
Amazing; very exciting!

**Comment by TomLav** [comment]
Amazing. Congratulations to the team(s)! And thank you.
